sinon/index.d.ts

```diff
@@ -427,7 +427,7 @@ declare namespace Sinon {
 
     // Utility overridables
     interface SinonStatic {
-        createStubInstance(constructor: any): SinonStub;
+        createStubInstance(constructor: any): any;
```

Comment:
@andy-ms  this method returns instantiated constructor where each method of instance is `SinonStub` so `SinonStub & T` is not the right thing